cypress/integration/hovercard-controller.spec.js

```diff
@@ -0,0 +1,150 @@
+context('HovercardController', () => {
+  beforeEach(() => {
+    cy.visit('/docs/hovercard-testing.html')
+  })
+
+  context('anchor is 0 to left of the viewport', () => {
+    it('hovercard is at top of the anchor', () => {
+      cy.get('#first').as('anchor')
+      cy.get('@anchor').trigger('mouseenter')
+      cy.get('div.hovercard:not(.is-hidden)').as('hovercard')
+      let anchorTop, widthHovercard, leftHovercard, leftSvg
```

Comment:
hay que ser consistentes con como llamadas las variables. En la primera (anchorTop) tienes el nombre del elemento primero (anchor) y la propiedad (top) despues. En las demas, tienes primero la propiedad (width/left) y luego el elemento (Hovercard, svg).

Creo que debería ser primero el elemento y luego la propiedad.

```suggestion
      let anchorTop, hovercardWidth, hovercardLeft, svgLeft
```